bin/ffscreencast

```diff
@@ -40,7 +40,7 @@ TIME=$(date +%H.%M.%S)
 NAME="Screencast ${DATE} at ${TIME}"
 
 # Where to save it
-DIR="${HOME}/Desktop"
+DIR="${HOME}/Videos/"
```

Comment:
Sorry but, No it's not mistaken. Is there something wrong?

Reply:
To be very clear: I think this is a good change to make, and might resolve https://github.com/cytopia/ffscreencast/issues/34 (or might not, perhaps that user doesn't keep a `~/Videos/` either).

This is a change in prior behavior from how the tool used to operate. It might be worth making a separate PR just for changing the default save directory, so @cytopia can either approve or reject it as they see fit, independent of the timer functionality your PR is adding.

Reply:
Videos directory is most likely available on all Desktop Linux distributions, and if it's not there that means most probably that user will neither have Desktop directory. (Like me)
BTW thanks.